test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -53,15 +53,16 @@ def hook(context):
     }
 
     # log tests scenario title on serverlog file
-    guiTestReportDir = os.environ.get("GUI_TEST_REPORT_DIR")
-    f = open(guiTestReportDir + "/serverlog.log", "a")
-    f.write(
-        str((datetime.now()).strftime("%H:%M:%S:%f"))
-        + "\tBDD Scenario: "
-        + context._data["title"]
-        + "\n"
-    )
-    f.close()
+    if os.getenv('CI'):
```

Comment:
don't we need the server log when running the tests locally?

Reply:
Locally `server log` can be viewed in the terminal. In my view, we don't need this while running tests locally.